06_puppy/mohitnag/syncstore.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"fmt"
+)
+
+// CreatePuppy creates a Puppy in syncstore
+func (s *SyncStore) CreatePuppy(p Puppy) error {
+	if _, ok := s.Load(p.ID); !ok {
+		s.Store(p.ID, p)
+		return nil
+	}
+	return fmt.Errorf("puppy with Id %d already exists", p.ID)
+}
+
+// ReadPuppy reads a Puppy from syncstore
+func (s *SyncStore) ReadPuppy(id uint32) (Puppy, error) {
+	if puppy, ok := s.Load(id); ok {
+		return puppy.(Puppy), nil
+	}
+	return Puppy{}, fmt.Errorf("puppy with Id %d does not exists", id)
+}
+
+// UpdatePuppy updates a Puppy in syncstore
+func (s *SyncStore) UpdatePuppy(p Puppy) error {
+
```

Comment:
unnecessary blank line.

Reply:
Lint dont allows me to delete. :|

Reply:
I don't see why you can't delete this line when all the other functions have this removed?

Reply:
Sorry i got confused with some other line. Removed now